ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -12,8 +12,9 @@ import { ButtonLink, HelpIcon } from '../../StyledComponents'
 import { getOtherGeneNames } from '../genes/GeneDetail'
 import Transcripts from './Transcripts'
 import VariantGenes, { LocusListLabels } from './VariantGene'
-import { getLocus, has37Coords, Sequence, ProteinSequence, TranscriptLink } from './VariantUtils'
+import { getLocus, has37Coords, Sequence, ProteinSequence, TranscriptLink, variantIntervalOverlap } from './VariantUtils'
 import { GENOME_VERSION_37, GENOME_VERSION_38, getVariantMainTranscript, SVTYPE_LOOKUP, SVTYPE_DETAILS, SCREEN_LABELS } from '../../../utils/constants'
+import SpliceOutlierLabel from './SpliceOutlierLabel'
```

Comment:
It would be more in keeping with the style of this folder to just define this component in this file directly